src/subdomains/voting/application/services/voting.service.ts

```diff
@@ -133,6 +170,19 @@ export class VotingService implements OnModuleInit {
     );
   }
 
+  private getUserVotes(user: User, cfpList: CfpInfo[]): CfpUserVote[] {
+    const userCfpVotes: CfpUserVote[] = [];
+    for (const [cfpId, vote] of Object.entries(user.vote)) {
```

Comment:
This doesn't work, because the user votes contain all votes from all previous votings. Logic needs to be inverted.
Loop through all CFPs in the list. Only if the user has voted for it, add it to the array.